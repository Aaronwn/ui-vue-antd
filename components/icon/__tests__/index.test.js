import { mount } from '@vue/test-utils'
import Icon from '..'

describe('Icon', () => {
  it('should render to a <i class="xxx"></i>', () => {
    const wrapper = mount({
      render (h) {
        return <Icon type='appstore' class='my-icon-classname' />
      },
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
