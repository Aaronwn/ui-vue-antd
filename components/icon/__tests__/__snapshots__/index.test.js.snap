// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon should render to a <i class="xxx"></i> 1`] = `<i class="anticon anticon-appstore my-icon-classname"></i>`;
